src/librados/RadosClient.cc

```diff
@@ -544,19 +544,22 @@ int librados::RadosClient::wait_for_osdmap()
   if (need_map) {
     Mutex::Locker l(lock);
 
-    utime_t timeout;
-    if (cct->_conf->rados_mon_op_timeout > 0)
-      timeout.set_from_double(cct->_conf->rados_mon_op_timeout);
+    mono_clock::duration timeout;
+    if (cct->_conf->rados_mon_op_timeout > 0) {
+      timeout = std::chrono::duration_cast<mono_clock::duration>(
```

Comment:
I think it's fine as-written in the code, but you might consider "using std::chrono" (and maybe even "using namespace std::chrono::literals"), it may make a readability difference (I don't find it too noisy as-is, though).

Reply:
Correct.